install failing on Windows 8

I've got Windows 8, Visual Studio 2015, Node 6.7.0, npm 3.10.3, node-gyp 3.5.0

I run `node install sodium` Downloads complete then this:

```
> sodium@2.0.1 install C:\Program Files\nodejs\node_modules\sodium
> node install.js --install

MS Version: 2015
Install Mode

C:\_dev\carwars-gl\node_modules\sodium>if not defined npm_config_node_gyp (node "C:\Us
dules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
Building the projects in this solution one at a time. To enable parallel build, please
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Platforms\x64\PlatformToolsets\
ll the required version of Windows SDK or change the SDK version in the project proper
ars-gl\node_modules\sodium\build\sodium.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed wit
gyp ERR! stack     at ChildProcess.onExit (C:\Users\eric.ehlers\AppData\Roaming\nvm\v6
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:2
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\eric.ehlers\\AppDat
ild"
gyp ERR! cwd C:\_dev\carwars-gl\node_modules\sodium
gyp ERR! node -v v6.7.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok
C:\_dev\carwars-gl\node_modules\sodium\install.js:288
            throw new Error(cmdLine + ' exited with code ' + code);
            ^

Error: node-gyp rebuild exited with code 1
    at ChildProcess.<anonymous> (C:\_dev\carwars-gl\node_modules\sodium\install.js:288
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
npm WARN carbuilder@1.0.0 No repository field.
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_m
npm ERR! node v6.7.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE

npm ERR! sodium@2.0.1 install: `node install.js --install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sodium@2.0.1 install script 'node install.js --install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the sodium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js --install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs sodium
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls sodium
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\_dev\carwars-gl\npm-debug.log
```

I had some time so I did some repairing on my Visual Studio and now getting a different error

```
C:\Users\eric.ehlers\AppData\Roaming\nvm\v6.7.0\node_modules\npm\node_modules\node-gyp\src\win_delay_load_hook.c(34): error C2373: '__pfnDliNotifyHook2': redefinition; different t
ype modifiers [C:\_dev\carwars-gl\node_modules\sodium\build\sodium.vcxproj]
  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\include\delayimp.h(134): note: see declaration of '__pfnDliNotifyHook2'
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\eric.ehlers\AppData\Roaming\nvm\v6.7.0\node_modules\npm\node_modules\node-gyp\lib\build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:106:13)
gyp ERR! stack     at ChildProcess.emit (events.js:191:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\eric.ehlers\\AppData\\Roaming\\nvm\\v6.7.0\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuil
d"
gyp ERR! cwd C:\_dev\carwars-gl\node_modules\sodium
gyp ERR! node -v v6.7.0
gyp ERR! node-gyp -v v3.3.1
gyp ERR! not ok
C:\_dev\carwars-gl\node_modules\sodium\install.js:288
            throw new Error(cmdLine + ' exited with code ' + code);
            ^

Error: node-gyp rebuild exited with code 1
    at ChildProcess.<anonymous> (C:\_dev\carwars-gl\node_modules\sodium\install.js:288:19)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:12)
npm WARN carbuilder@1.0.0 No repository field.
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "sodium"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE

npm ERR! sodium@2.0.1 install: `node install.js --install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sodium@2.0.1 install script 'node install.js --install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the sodium package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node install.js --install
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs sodium
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls sodium
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\_dev\carwars-gl\npm-debug.log
```

--- a/README.md
+++ b/README.md
@@ -122,6 +122,12 @@ At the moment Windows only supports dynamic linking so you must have the `libsod
 as `sodium.node`. This is done automatically by the install script, but if you move things around manually
 please don't forget to copy the DLL file as well.
 
+If you experience difficulty with the install even with a correctly set `msvs_version`, it may be worth trying:
+
+    npm install npm -g
+    
+to upgrade npm and its bundled version of node-gyp. 
+
 # Manual Build
 
 Node Sodium includes the source of libsodium, so the normal install will try to compile libsodium directly from source, using libsodium's own build tools.

